packages/origin-247-certificate/src/evented-certificate/repositories/CertificateCommand/CertificateCommandPostgres.repository.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+
+import { CertificateCommandEntity } from './CertificateCommand.entity';
+import {
+    CertificateCommandRepository,
+    NewCertificateCommand
+} from './CertificateCommand.repository';
+
+@Injectable()
+export class CertificateCommandPostgresRepository implements CertificateCommandRepository {
+    constructor(
+        @InjectRepository(CertificateCommandEntity)
+        private repository: Repository<CertificateCommandEntity>
+    ) {}
+
+    public async create(generation: NewCertificateCommand): Promise<CertificateCommandEntity> {
+        return await this.repository.save(generation);
+    }
+
+    public async getAll(): Promise<CertificateCommandEntity[]> {
+        return await this.repository.find();
+    }
+
+    public async getById(commandId: number): Promise<CertificateCommandEntity | null> {
```

Comment:
I like the approach where `get` always returns something or throws and `find` returns something or null.
https://szymonkrajewski.pl/the-practical-difference-between-findby-and-getby-in-repositories/